### ISSUE TITLE
Color contrast does not meet minimum requirements (selected facets on search results page)

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5346]
Sites identified: IJCCR
Page(s) identified: Media object page, section display

WCAG compliance level: AA

Description:

There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color blind or have low vision (vision loss equivalent to approx. 20/40 vision).

To meet the WCAG contrast requirement, you should have a contrast ratio of 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).

Note: IJCCR, MCO, and Archivo seem to use the same coloring when displaying select facets, so this is likely a problem across all sites.

See more details at [SiteImprove example](https://my2.siteimprove.com/Inspector/1167086/A11Y/Page?pageId=37971964269&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r69/failed/YnB1UzZtNjg=;NTQzNDU0YTg=).

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcjVnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--033afc14dfdd268e03fd4f475f1dad5e3d1526be/image.png)

# Done Looks Like
- [ ] Font color for the active facet text is changed from the light green to the darker standard link text used in Ramp (#333333)

**Comment by joncameron** [comment]
The light green causing contrast issues can be changed to the standard link color to meet requirements.

**Comment by joncameron** [comment]
Yep, this is OK with me! Makes sense that it would be black to match standard non-hyperlinked text.
